helpers/compile/plugins/fill-plugin/fillers/process.ts

```diff
@@ -126,6 +126,9 @@ export const process: NodeJS.Process = {
   setgid: noop(undefined),
   setgroups: noop(undefined),
   setuid: noop(undefined),
+  // See https://nodejs.org/api/process.html#processsourcemapsenabled
+  // Added in: v20.7.0
+  sourceMapsEnabled: false,
```

Comment:
What do you think about this @millsp, is that ok?

Reply:
Sounds good!